tests/erica_legacy/request_processing/erica_input/v2/test_grundsteuer_input_eigentuemer.py

```diff
@@ -83,3 +84,56 @@ def test_if_three_persons_and_verheiratet_false_then_raise_error(self):
         input_data = {"person": [person1, person2, person3], "verheiratet": False}
         with pytest.raises(ValidationError):
             Eigentuemer.parse_obj(input_data)
+
+    def test_if_all_persons_have_test_tax_id_number_then_raise_no_error(self):
+        persons = [Person(persoenliche_angaben=PersoenlicheAngaben(anrede="frau", name="daViella",
+                                                                                vorname="Gruella"),
+                                       adresse=Adresse(plz="79618", ort="Rheinfelden"),
+                                       steuer_id="04452317681", anteil=Anteil(zaehler="1", nenner="2")),
+                                Person(persoenliche_angaben=PersoenlicheAngaben(anrede="herr", name="Man",
+                                                                                vorname="Robin"),
+                                       adresse=Adresse(plz="79618", ort="Rheinfelden"),
+                                       steuer_id="09952417688", anteil=Anteil(zaehler="1", nenner="2")),
+                                Person(persoenliche_angaben=PersoenlicheAngaben(anrede="frau", name="Biest",
+                                                                                vorname="Bella"),
+                                       adresse=Adresse(plz="79618", ort="Rheinfelden"),
+                                       steuer_id="03352417692", anteil=Anteil(zaehler="1", nenner="2"))]
+        try:
+            Eigentuemer.parse_obj({"person": persons})
+        except ValidationError as e:
+            pytest.fail("parse_obj failed with unexpected ValidationError " + str(e))
```

Comment:
I thought at some point we decided against explicitly failing in tests? Because, at first I was a bit irritated^^